srcpkgs/vlc/template

```diff
@@ -36,7 +38,9 @@ makedepends="
  $(vopt_if v4l2 v4l-utils-devel) $(vopt_if smb samba-devel)
  $(vopt_if lua lua52-devel) $(vopt_if vaapi libva-devel)
  $(vopt_if vdpau libvdpau-devel) $(vopt_if x265 x265-devel)
- $(vopt_if svg librsvg-devel) $(vopt_if sndio sndio-devel)"
+ $(vopt_if svg librsvg-devel) $(vopt_if sndio sndio-devel)
+ $(vopt_if chromecast protobuf-devel) $(vopt_if chromecast protobuf)
+ $(vopt_if chromecast libmicrodns-devel)"
 depends="freefont-ttf hicolor-icon-theme desktop-file-utils"
 
 # Package build options
```

Comment:
The new option needs to be added to `build_options` so it can actually be used. Is there any reason not to add it as a default build option? It seems like a useful feature to have if you have a chromecast and having it enabled doesn't hurt if you haven't.

Reply:
I agree with it being a default build option. Without these options, the menu in VLC continuously  says "Scanning" without updating, so this should reduce confusion as well.

Thank you for the `build_options` pointer, fixed. I had this in an earlier diff locally and got mixed up making the PR.